### ISSUE TITLE
Error from creating a duplicate project returns non-user-facing error message

### ISSUE THREAD

**Issue by peterjmorgan** [phylum-dev/cli#508]
# Overview
When creating a new project that already exists, the returned error message is not a user-facing error message. 

# How To Reproduce
Steps to reproduce this behavior:
1. `phylum projects create ZZZ-example`
2. `phylum projects create ZZZ-example`
3. See error
```
[2022-06-30T02:55:58Z ERROR phylum] Execution failed: Dao Error:
     Conflict :                                                                                                                                                              Database(
        PgDatabaseError {
            severity: Error,
            code: "23505",
            message: "duplicate key value violates unique constraint \"projects_user_id_name_idx\"",
            detail: Some(
                "Key (user_id, name)=(00b28a50-1cb1-4bbe-bd24-f0c63e79cc7d, ZZZ-mono-main) already exists.",
            ),
            hint: None,                                                                                                                                                             position: None,
            where: None,
            schema: Some(
                "public",
            ),
            table: Some(
                "projects",
            ),
            column: None,
            data_type: None,
            constraint: Some(
                "projects_user_id_name_idx",
            ),
            file: Some(
                "nbtinsert.c",
            ),
            line: Some(
                656,
            ),
            routine: Some(
                "_bt_check_unique",
            ),
        },
    ): error returned from database: duplicate key value violates unique constraint "projects_user_id_name_idx"
❗ Error: Execution failed caused by: Dao Error:
```

# Expected Behavior
This error condition returns a formatted user-facing error explaining that a project with this name already exists.

**Comment by louislang** [comment]
I think that's right, @cd-work. I believe this issue will be fixed by a subsequent API release.

**Comment by kylewillmon** [comment]
Now that the new API release has happened, this issue has been fixed for all CLI clients version 3.7.0 and up.
